lib/Db/SubmissionMapper.php

```diff
@@ -87,26 +87,27 @@ public function findById(int $id): Submission {
 
 	/**
 	 * @param int $formId
-	 * @throws DoesNotExistException if not found
-	 * @return array
+	 * @param string $userId
+	 * @return bool
 	 */
-	public function findParticipantsByForm(int $formId): array {
+	public function hasSubmissions(int $formId, string $userId): bool {
```

Comment:
```suggestion
	public function hasFormSubmissionsByUser(int $formId, string $userId): bool {
```

What about including form and user in the function name? You could also please add some description of what it does to the PHPDoc comment.

Reply:
@Chartman123 As for the tests, I'm not sure if it makes sense to write tests for DB Mapper, since other classes of database work are not covered.